src/legacy/core_plugins/vis_default_editor/public/vis_type_agg_filter.ts

```diff
@@ -1,33 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
we no longer need this as aggFilter information is no longer present on schema, or anywhere on agg.